Makefile

```diff
@@ -108,24 +108,29 @@ endif
 
 ### Dependencies
 
+# Copied from OpenTelemetry Collector Makefile
+DOCKER_PROTOBUF ?= otel/build-protobuf:0.1.0
+PROTOC := docker run --rm -u ${shell id -u} -v${PWD}:${PWD} -w${PWD}/$(PROTO_INTERMEDIATE_DIR) ${DOCKER_PROTOBUF} --proto_path=${PWD}
+PROTO_INCLUDES := -I/usr/include/github.com/gogo/protobuf -I./opentelemetry-proto/ -Ipkg/tempopb/ -I./
```

Comment:
do we know what file are in this first -I param?  `/usr/include/github.com/gogo/protobuf` ?

Reply:
This was copied over from Otel collector Makefile, I don't think we need those imports, checking.